### PR TITLE
REad me update for the custom check

### PR DESCRIPTION
**Pull request by amit-chandak-unskript**
## Description
Read me update for adding custom check

### CODE REVIEW

File: README_extending_docker.md [hunk 1]
```diff
@@ -93,4 +93,48 @@ Lets consider two scenarios as starting point for extending Awesome docker
...
+
+./Awesome-CloudOps-Automation/bin/unskript-add-check.sh -t <Check type> -n <short name for the check, separated by _> -d <description of the check>
+
+```
+The above command will create the template .py and pytest files. For eg:
+```
+(py396) amits-mbp-2:custom-checks amit$ ls -l actions/aws_list_public_sg/
+total 24
+-rw-r--r--  1 amit  staff     0 Sep 25 17:42 __init__.py
+drwxr-xr-x  5 amit  staff   160 Sep 25 21:09 __pycache__
```
Comment (jayasimha-raghavan-unskript) [inline README_extending_docker.md:122]: Remove __pycache__ line

File: README_extending_docker.md [hunk 2]
```diff
@@ -93,4 +93,48 @@ Lets consider two scenarios as starting point for extending Awesome docker
 
 ### How to create Custom Actions
 
-You can refer to [this link](https://docs.unskript.com/unskript-product-documentation/actions/create-custom-actions) on how to create custom Action
+You can create custom action on your workstation using your editor. Please follow the steps below to setup your workstation:
+
```
Comment (jayasimha-raghavan-unskript) [inline README_extending_docker.md:97]: You can also refer them to https://realpython.com/intro-to-pyenv/#:~:text=Activating%20Your%20Versions,-Now%20that%20you&text=python%2Dversion%20file%20in%20your,environment%20will%20automatically%20be%20activated. if they want to use pvenv

**Review by jayasimha-raghavan-unskript — APPROVED** [review]
Please see if you can take care of the comments.

File: README_extending_docker.md (hunk 2 above)
Comment (amit-chandak-unskript) [reply README_extending_docker.md:97]: I dont want to document multiple ways, if the user knows it can be accomplished via pyenv, let them use that.

**Pull request merged**